src/components/dropdown.ts

```diff
@@ -0,0 +1,27 @@
+import {Component, View} from 'angular2/angular2';
+
+@Component({
+  selector: 'uib-dropdown',
```

Comment:
Did we decide in the prefix already? I was under the impression that we want to cut ties to angular-ui, no?